changelog.d/7848.misc

```diff
@@ -0,0 +1 @@
+Remove redundant `retry_on_integrity_error` wrapper for event persistence code.
```

Comment:
What makes this redundant? Is essentially the same thing done right now anyway?

Reply:
well, see the PR description.

Reply:
So less redundant, and more not useful. OK.